src/platform/packages/shared/kbn-securitysolution-ecs/src/entity/index.ts

```diff
@@ -7,13 +7,11 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-// TODO: Asset Inventory - This file is a placeholder for the ECS schema that will be used in the Asset Inventory app
 export interface EntityEcs {
```

Comment:
@JordanSh I see the Threat Hunting approval is required for this change as it's a codeowner for the `kbn-securitysolution-ecs` package, but I guess it would make sense to include our team to be a co-owner of the  `src/platform/packages/shared/kbn-securitysolution-ecs/src/entity` path if the @elastic/security-threat-hunting doesn't oppose to this

Reply:
Sounds good to me